README.md

```diff
@@ -1,88 +1,127 @@
-## Travel Tracker
+# Travel Tracker
 
-### Overview
+> Travel Tracker is a web application which manages and tracks different trips for users and travel agents, by consuming and posting to an external API.
 
-Travel Tracker is an application which manages and tracks different trips for
-users and travel agents, by consuming and posting to an external API.
-This is a project made by [Joshua Everett](https://github.com/josh-everett01).
-This project implements HTML / CSS for the main skeleton design and styling, and
-uses JavaScript to authorize user login, make calls to the API, and to populate
-travelers / trips information on the user dashboard.
+[Travel Tracker Live Link](https://josh-everett01.github.io/travel_tracker/)
+This project was created by [Joshua Everett](https://github.com/josh-everett01)
 
-#### Login
+## Table of contents
 
-**Upon opening / navigating to the project:**
-The user sees a login screen which
-accepts a username based on either a traveler or an agent of a travel agency.
-When a traveler logs in a fetch call is made to the API to verify the travelers
-ID against the ID's in the Array of travelers.
+- [Overview](#overview)
+- [Screenshot](#screenshot)
+- [Setup](#setup)
+- [Tech Stack](#tech-stack)
+- [Learning Objectives](#learning-objectives)
+- [Credits](#credits)
+- [Contact](#contact)
 
-#### Traveler Dashboard
+# Overview
 
-**If a traveler logs in they see a dashboard page showing them:**
+Travel Tracker utilizes the fetch API to gather data from multiple endpoints. A user is able to login as either an agent or traveler, and then they are able to view trips, make trip requests, and approve / delete trip requests depending on who is logged in.
```

Comment:
In here mention and reference the API you used, also should be noted you did it using an agile workflow and a pull request process (Sr. Dev, Jr. Dev paradigm).  

This sort of thing isn't useful for an open source project or whatever, but since this is a "Resume Project" the workflows and project management should be mentioned. 

Probably should note the project is based on the open source Turing school curriculum and link it